bin/otu_to_tax_name.py

```diff
@@ -51,23 +46,24 @@ def main():
 
     # Read in otus file data
     try:
-        otu_fnh = pd.read_csv(args.otu_id_fp, sep="\n")
+        with open(args.otu_id_fp, "rU") as inf:
+            otu_ids = [line.strip() for line in inf.readlines()]
```

Comment:
This won't handle multi-column files. The csv module will be the simplest way to handle it:
`csvr = csv.reader(inf, delimiter="\t")`
`otu_ids = [line[0] for line in csvr]`